automation/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM centos:7
+ENV container docker
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == \
```

Comment:
Why is this needed? Please add some explanation as a comment.

Reply:
Done